### ISSUE TITLE
AddHandler causing download file in Localhost/Laragon

### ISSUE THREAD

**Issue by cacacobra** [leokhoa/laragon#538]
On my cPanel Server it includes the code below in my htaccess, so when i run my system in Laragon/Localhost it causes download file index.

Is there any way to avoid this without changing the htaccess?

Inside Apache configs maybe?

 \# php -- BEGIN cPanel-generated handler, do not edit 
 \# Defina o pacote “ea-php70” como a linguagem padrão de programação “PHP”.
<IfModule mime_module>
  AddHandler application/x-httpd-ea-php70 .php .php7 .phtml
</IfModule>
 \# php -- END cPanel-generated handler, do not edit

**Comment by RobiNN1** [comment]
I don't think it can be fixed without changing. It downloads file because x-httpd-ea-php70 doesn't exist in laragon.

**Comment by cacacobra** [comment]
> I don't think it can be fixed without changing. It downloads file because x-httpd-ea-php70 doesn't exist in laragon.

Is there any possibility to configure Laragon to accept it?

**Comment by RobiNN1** [comment]
I remembered small trick I've used for php per project setup, try this:

Add this to the end of your apache config before `IncludeOptional ...`, in my case it's `C:/laragon/bin/apache/httpd-2.4.41-win64-VC14/conf/httpd.conf` 

And of course set correct path to the your php

```
ScriptAlias /php70 "C:/laragon/bin/php/php-7.0.33-Win32-VC14-x64"
Action application/x-httpd-ea-php70 /php70/php-cgi.exe
<Directory "C:/laragon/bin/php/php-7.0.33-Win32-VC14-x64">
    AllowOverride None
    Options None
    Require all denied
    <Files "php-cgi.exe">
        Require all granted
    </Files>
</Directory>
```

**Comment by alxndr-w** [comment]
I think, @RobiNN1 did offer a practicable workaround / solution to this. @cacacobra: If you can remember, if this solved the issue, let us know.
